docs/snippets/common/storybook-preview-auto-docs-custom-docs-container.js.mdx

```diff
@@ -0,0 +1,31 @@
+```js
+// .storybook/preview.js
+
+import { DocsContainer } from '@storybook/blocks';
+
+const ExampleContainer = (props) => {
```

Comment:
It'd be great to demo at least some of the useful `props` you're given.

E.g., I've personally used this feature to avoid defining the title (pre "auto title") in both the CSF meta and the MDX (as an h1).

Reply:
Or, alternatively, show how you can access what's in the context via the hook?

Reply:
I'll provide an example in a bit...

Reply:
Decided it was best to not to be prescriptive when providing an example:

```jsx
import * as React from "react";
import { DocsContainer } from "@storybook/blocks";

const ExampleContainer = ({ children, ...props }) => {
  console.log("DocsContainer props", props);

  return <DocsContainer {...props}>{children}</DocsContainer>;
}

export const parameters = {
  actions: { argTypesRegex: "^on[A-Z].*" },
  controls: {
    matchers: {
      color: /(background|color)$/i,
      date: /Date$/,
    },
  },
  docs: {
    container: ExampleContainer,
  },
};
```